.docker/scripts/deactivate

```diff
@@ -0,0 +1,3 @@
+if [ "$VENV_DIR" != "" ]; then
+  deactivate
```

Comment:
Should this be `. ${VENV_DIR}/bin/deactivate` ?

Reply:
Not necessary. If a virtualenv has been activated, then its bin directory is on the PATH. We shouldn't make assumptions about where that is, if we don't need to.